Gemfile

```diff
@@ -12,6 +12,9 @@ gemspec
 
 gem "inventory_refresh", :git => "https://github.com/ManageIQ/inventory_refresh", :branch => "master"
 
+# Until https://github.com/ErwinM/acts_as_tenant/pull/192 is released
+gem "acts_as_tenant", :git => "https://github.com/ErwinM/acts_as_tenant", :branch => "master"
```

Comment:
`hair_trigger` was blowing up trying to access `Rails.application` [here](https://github.com/jenseng/hair_trigger/blob/master/lib/hair_trigger.rb#L25-L27), it had logic to handle the non-Rails case but since we were defining a `Rails` stub it was blowing up.